Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -45,7 +45,7 @@ public class FocusProvider : BaseCoreSystem,
             focusIndividualCompoundCollider = profile.FocusIndividualCompoundCollider;
         }
 
-        private readonly HashSet<PointerData> pointers = new HashSet<PointerData>();
+        private readonly Dictionary<uint, PointerData> pointers = new Dictionary<uint, PointerData>();
```

Comment:
Was this change intended for this PR too?
If so, do you happen to have perf measurements before / after? Looking at the code, I'd expect this to be better, but was wondering to be sure.

Reply:
It was, but i forgot to add it to the description. :/

i dont have before / after. anecdotally, the partner we tested this change for states it is an improvement.

Reply:
added comment to description